devito/finite_differences/differentiable.py

```diff
@@ -228,9 +228,13 @@ def __neg__(self):
         return Mul(sympy.S.NegativeOne, self)
 
     def __eq__(self, other):
-        return super(Differentiable, self).__eq__(other) and\
-            all(getattr(self, i, None) == getattr(other, i, None)
-                for i in self.__rkwargs__)
+        if super(Differentiable, self).__eq__(other) is not True:
```

Comment:
can you not just do `super(Differentiable, self).__eq__(other) is True and\

Reply:
Thanks @mloubout , seems to work after a quick local test.